tests/conftest.py

```diff
@@ -98,7 +98,8 @@ def compile_and_run(device, reset_torch_dynamo, request):
             outputs_after = run_model(m, inputs)
             end = time.perf_counter() * 1000
             comp_runtime_metrics = {"success": True, "run_time": round(end - start, 2)}
-            option._out_fx_graphs[0].print_tabular()
+            if option.verbose:
+                option._out_fx_graphs[0].print_tabular()
```

Comment:
This change doesn't seem relevant to this particular commit.

Reply:
Yeah not relevant to this specific PR, but printing should be optional depending on verbosity.

Reply:
Not relevant change has to be moved out to a separate PR. I know its annoying, but thats the eng culture to project. Please do so.

I will do better myself too.